### PR TITLE
SSCS-10286 Add Handling Error State

### PR DESCRIPTION
**Pull request by bensonlievoco**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10286

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java [hunk 1]
```diff
@@ -595,7 +594,7 @@ public SchedulingAndListingFields getSchedulingAndListingFields() {
 
     @JsonIgnore
     public JointParty getJointParty() {
-        if (isNull(jointParty)) {
+        if (jointParty == null) {
```
Comment (dattipoeHMCTS) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java:597]: Why was this changed?

**Review by MohammedHMCTS — APPROVED** [review]
Looks good to me apart from Dels comment that just needs checking.

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java (hunk 1 above)
Comment (bensonlievoco) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java:597]: @nickwest1 given a comment that in the previous branch, he suggest align using == null / isNull
